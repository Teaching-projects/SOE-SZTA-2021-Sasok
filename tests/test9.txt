15
3
3
14
0
2