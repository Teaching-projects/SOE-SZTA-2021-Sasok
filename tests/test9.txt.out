Első egység HP-ja: Első egység DMG-je: Első egység AS-je: Második egység HP-ja: Második egység DMG-je: Második egység AS-je: 
A Csata elkezdődött! Harcos és Sámán megtámadtak egymást. életük: Harcos 15, Sámán 11
Sámán megtámadta Harcos, így Harcos élete - 15
Harcos megtamádta Sámán, így Sámán élete - 8
Sámán megtámadta Harcos, így Harcos élete - 15
Harcos megtamádta Sámán, így Sámán élete - 5
Sámán megtámadta Harcos, így Harcos élete - 15
Sámán megtámadta Harcos, így Harcos élete - 15
Harcos megtamádta Sámán, így Sámán élete - 2
Sámán megtámadta Harcos, így Harcos élete - 15
Harcos megtamádta Sámán, így Sámán élete - -1
Sámán megtámadta Harcos, így Harcos élete - 15

Harcos GYOZEDELMESKEDETT!
